dmoj/citest.py

```diff
@@ -17,8 +16,10 @@ def ci_test(executors_to_test, overrides, allow_fail=frozenset()):
     failed = False
     failed_executors = []
 
+    load_executors()
```

Comment:
Are you calling this for the side-effect of loading all executors into `sys.modules`? Feels kinda weird since it prints `Self-testing executors` and does a bunch of other things. Might be better to refactor the loading logic in `load_modules` into a function and call that, then don't load these modules.